### PR TITLE
Add support for IP discovery profile resource

### LINKED ISSUE #414: Provide support to create ip discovery profiles
Currently, the provider supports discovery providers as data source and also to use them on a segment (although its an undocumented feature, its not shown here: https://www.terraform.io/docs/providers/nsxt/r/policy_segment.html)

We should also provide the ability to create an IP Discovery Policy (same for the other segment policies I guess) to manage the policies in Terraform. As far as I can tell, the SDK already supports that.

### PR DESCRIPTION
**Pull request by ksamoray**
Signed-off-by: Kobi Samoray <ksamoray@vmware.com>

Fixes #414

### CODE REVIEW

File: nsxt/resource_nsxt_policy_ip_discovery_profile_test.go [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+			},
+			{
+				Config: testAccNsxtPolicyIPDiscoveryProfileMinimalistic(),
+				Check: resource.ComposeTestCheckFunc(
+					testAccNsxtPolicyIPDiscoveryProfileExists(accTestPolicyIPDiscoveryProfileCreateAttributes["display_name"], testResourceName),
+					resource.TestCheckResourceAttr(testResourceName, "description", ""),
+					resource.TestCheckResourceAttrSet(testResourceName, "nsx_id"),
+					resource.TestCheckResourceAttrSet(testResourceName, "path"),
+					resource.TestCheckResourceAttrSet(testResourceName, "revision"),
+					resource.TestCheckResourceAttr(testResourceName, "tag.#", "0"),
```
Comment (annakhm) [inline nsxt/resource_nsxt_policy_ip_discovery_profile_test.go:108]: Please also ensure that other attributes are set back to defaults

**Comment by annakhm** [comment]
Please add documentation files - under /website/docs/r.
Otherwise looks good!

**Review by annakhm — APPROVED** [review]

**Comment by salv-orlando** [comment]
@annakhm Kobi pushed a new update to this PR after your approval, can you review again?

File: website/docs/r/policy_ip_discovery_profile.html.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+This resource is applicable to NSX Global Manager, NSX Policy Manager and VMC.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_ip_discovery_profile" "ip_discovery_profile" {
+  description  = "ip discovery profile provisioned by Terraform"
+  display_name = "ip_discovery_profile1"
+
+  arp_nd_binding_timeout         = "20"
```
Comment (annakhm) [inline website/docs/r/policy_ip_discovery_profile.html.markdown:21]: please remove quotes around integers and booleans, only strings should be quoted

**Review by annakhm — APPROVED** [review]

**Pull request merged**